backend.tf

```diff
@@ -1,9 +1,9 @@
 terraform {
   backend "s3" {
     encrypt        = true
-    bucket         = "playground-terraform-state-storage"
+    bucket         = "cisa-cool-terraform-state"
     dynamodb_table = "terraform-state-lock"
-    profile        = "playground"
+    role_arn       = "arn:aws:iam::608004238745:role/TerraformStateAccess"
```

Comment:
I can't remember what we decided.  Weren't we keeping all the ARNs inside our aws configs, and referencing the role names?  (See the "roles" tab of the "CISA COOL AWS Accounts" spreadsheet.  

This allows us to use roles without checking in our account numbers to version control.

Reply:
> I can't remember what we decided. Weren't we keeping all the ARNs inside our aws configs, and referencing the role names? (See the "roles" tab of the "CISA COOL AWS Accounts" spreadsheet.
> 
> This allows us to use roles without checking in our account numbers to version control.

I had it pointing to profiles before, but I thought we decided to use ARNs instead.  See, for example, [this commit](https://github.com/cisagov/pca-terraform/pull/7/commits/2a0294214e5b921330c8b4afe7a85ea6968d64ea).

@dav3r may be able to shed some light on this.  His memory tends to be better than mine.

Reply:
Using ARNs means one less layer of obfuscation, and one can look at the code in the repo and see what is going on without any knowledge of the way that we set up our boto3 profiles.

I think what makes the most sense is to use a profile to configure the backend (since variables are not allowed in that configuration block and we don't want actual ARN values out in the open) but use ARNs specified in variables (and hence in a `*.tfvars` file that is not committed) to configure the providers.

See commit 622b7e0.